applications/produce_array_config.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+    This application is a simple example of how to produce sim_telarray config \
+    files for a given array.
+
+    All the input required has to be given as a yaml file by the command \
+    line argument array_config.
+
+    The required entries in the array_config file are:
+
+    'site': South or North
+
+    'layoutName': name of a valid layout array.
+
+    'modelVersion': name of a valid model version.
+
+    'default': telescope model names to be assigned to each telescope size by default. \
+    It must contain entries for 'LST' and 'MST' (and 'SST' in case of South site).
+
+    As optional data, specific telescope models can be assigned to individual telescopes. \
+    This is done by the entries with the name of the telescope (as used by the layout \
+    definition, ex. L-01, M-05, S-10).
+
+    Each telescope model can be set in two ways.
+
+    a) A single str with the name of telescope model.
+    Ex. 'M-05': 'NectarCan-D'
+
+    b) A dict containing a 'name' key with the name of the telescope model and further keys \
+    with model parameters to be changed from the original model.
+    Ex.:
+
+    'M-05': {
+            'name': 'NectarCam-D',
+            'fadc_pulse_shape': 'Pulse_template_nectarCam_17042020-noshift.dat',
+            'discriminator_pulse_shape': 'Pulse_template_nectarCam_17042020-noshift.dat'
+        }
+
+    This is an example of the content of an array_config file.
+
+    site: North,
+    layoutName: Prod5
+    modelVersion: Prod5
+    default:
+        LST: 'D234'  # Design model for the LSTs
+        MST: FlashCam-D  # Design model for the MST-FlashCam
+    L-01: '1'  # Model of L-01 in the LaPalma site.
+    M-05:
+        name: NectarCam-D
+        fadc_pulse_shape: Pulse_template_nectarCam_17042020-noshift.dat  # Parameters to be changed
+        discriminator_pulse_shape: Pulse_template_nectarCam_17042020-noshift.dat
+
+    Command line arguments
+    ----------------------
+    label (str, optional)
+        Label to identify the output files/directories.
+    array_config (str, required)
+        Path to a yaml file with the array config data.
+    verbosity (str, optional)
+        Log level to print (default=INFO).
+
+    Example
+    -------
+    North - Prod5, simple example
+
+    Runtime < 1 min.
+
+    .. code-block:: console
+
+        python applications/produce_array_config.py --label test --array_config \
+        data/test-data/arrayConfigTest.yml -v DEBUG
+
+    All the model files can be found in simtools-output/test/model/
+
+'''
+
+import logging
+import argparse
+
+import simtools.util.general as gen
+from simtools.model.array_model import ArrayModel
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(
+        description=(
```

Comment:
Is this description correct? 'calculate PSF'?

Reply:
No, it is a leftover from the 'template' that I used, It is fixed now.